firebase-messaging/CHANGELOG.md

```diff
@@ -1,12 +1,21 @@
 # Unreleased
-* [deprecated] Deprecated FCM upstream messaging. See the
-  [FAQ](https://firebase.google.com/support/faq#fcm-23-deprecation) for more
-  details.
+
```

Comment:
Isn't there not supposed to be a blank line here?

Reply:
Removed the blank line